### PR TITLE
Add German localization

### PR DESCRIPTION
**Pull request by vitusortner**
This PR adds German localization to the app. To fully localize it, I extracted some strings that weren't ready for localization yet.

| Before | After |
| --- | --- |
| Screenshot 2022-12-27 at 12 40 17 | Screenshot 2022-12-27 at 12 34 23 |
| Screenshot 2022-12-27 at 12 40 19 | Screenshot 2022-12-27 at 12 34 27 |
| Screenshot 2022-12-27 at 12 39 13 | Screenshot 2022-12-27 at 12 34 00 |
| Screenshot 2022-12-27 at 12 39 39 | Screenshot 2022-12-27 at 12 34 12 |

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
             android:enabled="true"
             android:exported="true"
             android:icon="@drawable/baseline_highlight_24"
-            android:label="Flashlight"
+            android:label="@string/tile_title"
```
Comment (vitusortner) [inline app/src/main/AndroidManifest.xml:35]: to be able to localize the tile's title.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -18,13 +18,14 @@
     <string name="add_title_in_device_control">Add title in Device Control</string>
     <string name="explanation_of_setting_items">Explanation of setting items</string>
 
-    <string name="title">Explanation of setting items</string>
```
Comment (vitusortner) [inline app/src/main/res/values/strings.xml:21]: removed as it was a duplicate of `explanation_of_setting_items`.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -2,7 +2,6 @@
     <string name="app_name">Flashlight Tiramisu</string>
     <string name="add_qs_title">Add QS Title</string>
     <string name="settings">Settings</string>
-    <string name="add_flashlight_title_in_device_control">Add flashlight title in Device Control</string>
```
Comment (vitusortner) [inline app/src/main/res/values/strings.xml:5]: was unused

**Comment by polodarb** [comment]
Thank you for your contribution

**Pull request merged**